stories/H2.story.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { H2, Colors } from "../src/index";
+
+storiesOf("HeadingTwo", module)
```

Comment:
> I also wonder if all these typographies belong better under one "typography" story header, but we can do that in a PR later if we both agree

I dont know, lets decide. I can also add it there but since its components maybe it needs also to be their own stories?

Reply:
Yeah, I mean like under a `Typography` subtree or something, I think we can do it by just adding `Typography/` to the start of the story names https://storybook.js.org/docs/basics/writing-stories/#nesting-stories

Reply:
I am wondering if we want the structure of the storybook to be different that the structure of the `src` in our project.

Reply:
Yeah true, just worried that it will be messy if they arent grouped together, lets wait and see :)

Reply:
i dont mind if we group them together or not. But maybe we want to group them inside the `src`? 
anyway lets wait and see. no big deal to change this around 👍